src/routes/InstancesPage/CreateInstanceModal.js

```diff
@@ -14,14 +14,15 @@ import {
   ToggleGroupItem,
 } from '@patternfly/react-core';
 
-import { regionOptions } from '../../utils/region';
 import SelectSingle from '../../components/SelectSingle';
 import useAnalytics from '../../hooks/useAnalytics';
+import { useCloudRegions } from '../../hooks/apis/useCloudRegions';
+import { AWS_PROVIDER } from '../../utils/cloudProvider';
 
 const defaultFormValues = {
   name: '',
-  cloud_provider: 'aws',
-  region: 'us-east-1',
+  cloud_provider: AWS_PROVIDER,
+  region: '',
```

Comment:
Is now the default value the first region, in alphabetical order?

I think `us-east-1` should stay as default.

Reply:
➕